src/fs-plus.coffee

```diff
@@ -60,6 +60,21 @@ fsPlus =
       return "#{fsPlus.getHomeDirectory()}#{relativePath.substring(1)}"
     return relativePath
 
+  # Public: Convert an absolute path to tilde path for linux and mac:
+  # /Users/username/dev => ~/dev
+  #
+  # pathToTildify - The {String} containing the full path.
+  #
+  # Returns a tildified path {String}.
+  tildify: (pathToTildify) ->
+    if process.platform is 'win32'
```

Comment:
One more thing (sorry, didn't catch this the first time):
`return pathToTildify if process.platform is 'win32'`

You can probably also do that for the one on line 74.